src/main/java/org/openrewrite/java/spring/boot2/ConvertToSecurityDslVisitor.java

```diff
@@ -181,17 +182,17 @@ public boolean isApplicableTopLevelMethodInvocation(J.MethodInvocation m) {
         return false;
     }
 
-    private boolean isApplicableCallCursor(Cursor c) {
-        if (c != null && c.getValue() instanceof J.MethodInvocation) {
-            J.MethodInvocation inv = c.getValue();
-            if (!TypeUtils.isOfClassType(inv.getType(), securityFqn)) {
-                return true;
-            }
-            if (new MethodMatcher("org.springframework.security.config.annotation.web.configurers.AbstractHttpConfigurer disable()").matches(inv)) {
-                return true;
-            }
+    private boolean isApplicableCallCursor(@Nullable Cursor c) {
+        if (c == null) {
+            return false;
         }
-        return false;
+
+        if (!(c.getValue() instanceof J.MethodInvocation)) {
+            return false;
+        }
+
+        J.MethodInvocation inv = c.getValue();
+        return !(isAndMethod(inv) && TypeUtils.isOfClassType(inv.getType(), securityFqn)) && !isDisableMethod(inv);
```

Comment:
We're also calling `isOfClassType` in `isAndMethod`, so I'll go ahead and remove that before a merge.
```suggestion
        return !isAndMethod(inv) && !isDisableMethod(inv);
```

Reply:
Good catch! 

I had to move it back during the refactoring as initially I was attempting to move toward a `MethodMatcher` based solution which wasn't checking the return type. With Spring Security there are a few instances of `and` that I found that don't inherit that method from the common interface, so I had to double back on that and missed getting it here. Oops!